benchmarks/src/main/scala/zio/redis/BenchmarksUtils.scala

```diff
@@ -16,5 +18,6 @@ trait BenchmarksUtils {
 }
 
 object BenchmarksUtils {
-  private final val Layer = Logging.ignore >>> RedisExecutor.local.orDie
+  private final val Codec: ULayer[Has[Codec]] = ZLayer.succeed(StringUtf8Codec)
```

Comment:
I'd "inline" this into `Layer` definition if possible.

Reply:
Done